modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -845,6 +804,50 @@ void ParallelCoordinates::updateBrushing() {
     brushingAndLinking_.sendFilterEvent(brushedID);
 }
 
+void ParallelCoordinates::updateAxesLayout() {
+    auto firstVisible = true;
+    for (auto &p : axisVector_) {
+        auto &prop = std::get<1>(p);
+        // Place labels on the left side of axis for the first and right side for the others
```

Comment:
`// Place labels on the left side of the first axis and on the right for all other axes`